articles/azure-functions/functions-dotnet-class-library.md

```diff
@@ -267,7 +267,6 @@ public static class QueueFunctions
 {
     // Queue trigger
     [FunctionName("QueueTrigger")]
-    [StorageAccount("AzureWebJobsStorage")]
```

Comment:
The `StorageAccount` attribute was already added to the class, so no need to apply it on the method.